cfgov/jinja2/v1/_includes/organisms/item-introduction.html

```diff
@@ -34,9 +34,7 @@
 {% set has_authors = page.authors.exists() %}
 
 <div class="o-item-introduction">
-    {% if value.category is string %}
```

Comment:
I think there might be a problem with this change, as category URLs don't seem to render properly.

I tried following the instructions above on both a new local database as well as with a production dump. When the category link gets generated on the page, you get something like

http://localhost:8000/testing/#?filter**{{ index }}**_categories=us-supreme-court

(Emphasis added.) On [the example page](http://localhost:8000/policy-compliance/rulemaking/final-rules/2013-real-estate-settlement-procedures-act-regulation-x-and-truth-lending-act-regulation-z-mortgage-servicing-final-rules/) that your screenshot is from above, the category URL is

http://localhost:8000/policy-compliance/rulemaking/final-rules/?filter**{{ index }}**_categories=final-rule

To me it looks like the variable `index` that is getting referred to is never being set anywhere. Maybe it should be `loop.index`? I'm not quite sure how that variable is used.